src/modules/addComments.js

```diff
@@ -0,0 +1,61 @@
+import displayComments from './displayComments.js';
+
+const addComments = () => {
+  const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/IR7MovTrVQtBQVyC6UTK/comments';
+  const CommentsForm = document.querySelector('.form-comments');
+
+  const postComment = async (data) => {
+    try {
+      const response = await fetch(url, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(data),
+      });
+      if (response.ok) {
+        const okstatus = document.createElement('div');
+        okstatus.setAttribute('id', 'message');
+        okstatus.innerHTML = ` <p>Comment Added</p>
+        `;
+        CommentsForm.appendChild(okstatus);
+        setTimeout(() => {
+          document.getElementById('message').innerHTML = '';
+        }, 3000);
+      }
+
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+    } catch (error) {
+      // eslint-disable-next-line no-console
```

Comment:
- [ ] For best practice reasons and since it is a common practice in real working environment, I suggest fixing this listing issue instead of disabling linter checks here. Kindly find a way to resolve this without obstructing linter checks :pray: